src/Lykke.Job.ServicesMonitoring/TriggerHandlers/CheckAllServices.cs

```diff
@@ -5,34 +5,38 @@
 using System.Net;
 using System.Text;
 using System.Threading.Tasks;
+
 using Common;
 using Common.Log;
-using Lykke.JobTriggers.Triggers.Attributes;
-using Lykke.Job.ServicesMonitoring.Core;
-using Lykke.Job.ServicesMonitoring.Core.Domain;
+
 using Lykke.Job.ServicesMonitoring.Core.Domain.Monitoring;
-using Lykke.Job.ServicesMonitoring.Services;
+using Lykke.Job.ServicesMonitoring.Settings;
+using Lykke.Job.ServicesMonitoring.Settings.JobSettings;
+using Lykke.Job.ServicesMonitoring.Settings.SlackNotifications;
 using Lykke.Job.ServicesMonitoring.Models;
+using Lykke.Job.ServicesMonitoring.Services;
+using Lykke.JobTriggers.Triggers.Attributes;
+using Lykke.SettingsReader;
 
 namespace Lykke.Job.ServicesMonitoring.TriggerHandlers
 {
     public class CheckAllServices
     {
         private const string SlackMonitorChannel = "Monitor";
         private readonly IServiceMonitoringRepository _serviceMonitoringRepository;
-        private readonly HostToCheck[] _hostsToCheck;
-        private readonly SlackIntegrationSettings _slackIntegrationSettings;
+        private readonly IReloadingManager<ServiceMonitoringSettings> _serviceMonitoringSettings;
```

Comment:
Better avoid using IReloadingManager in case of canstant settings (not a connection string).
So you need to use .CurrentValue on IReloadingManger to get actual settings velut to store it in readonly fields